source/_includes/installation/operating_system.md

```diff
@@ -242,7 +242,8 @@ _All these can be extended if your usage calls for more resources._
     2. Select Type "Linux" and Version "Linux 2.6 / 3.x / 4.x (64-bit)"
     3. Select "Use an existing virtual hard disk file", select the unzipped VDI file from above
     4. Edit the "Settings" of the VM and go "System" then "Motherboard" and select "Enable EFI"
-    5. Then go to "Network" "Adapter 1" choose "Bridged Adapter" and choose your Network adapter
+    5. Edit the "Settings" of the VM and go "System" then "Processor" and set 2 CPU cores
```

Comment:
Is that a bug? Or by design?

Reply:
this is an improvement.  Without this, the virtual box will not start.

https://community.home-assistant.io/t/virtualbox-haos-ova-8-4-vdi-wont-boot/447485

Reply:
Cool, but is that by design or a bug?
As in, I understand that that is working, but is that something that needs to be documented or fixed? Has this been reported?

Reply:
@almirus as I've noted in the comments above, it depends on the exact configuration. Therefor, instead of simply suggesting two, I suggest to create a note that people should try with 1 or 2 CPUs and see what is working best for them.

Can you update the PR accordingly?

Reply:
It is unlikely that people are still in that dialog when they experience the problem. I'd prefer an info box below, similar to the discard option (see https://www.home-assistant.io/installation/linux#hypervisor-specific-configuration).

Also, you'd need to renumber the step 6 Audio below anyways.

Reply:
> @agners `Edit the "Settings" of the VM and go "System" then "Processor" and set 2 CPU cores (Optional, if only problems appear) ` Is it ok?

yeah well and good , what about config tho ?/